README.md

```diff
@@ -58,12 +59,15 @@ The URL printed under Location is the file that you want.
 
 # For mod authors
 If you want the package detection to work, make sure you attach packages to the Github release and mind these patterns:
- * a windows exe package name should end with "exe"
- * a windows zip package name should start with "win" and end with "zip"
- * a linux package name should start with "lin"
- * an osx package name should start with "osx"
+ * universal zip package name should end with ".zip"
+ * windows exe package name should end with ".exe"
+ * windows zip package name should start with "win-" and end with ".zip"
+ * linux package name should start with "lin-"
+ * macOS package name should start with "osx-"
  
-For an example, check [a release](https://github.com/Gibberlings3/Tweaks-Anthology/releases/tag/Beta_5) of Tweaks-Anthology.
+WARNING: for uzp, if there are multiple .zip files (except those with 'win-', 'osx-', 'lin-' prefixes) in the release, behaviour is undefined.
```

Comment:
This warning would better fit in the main section above, since it's more for users than modders. It will also unbreak the flow of thought, since Tweaks Anthology isn't an example of this.

An example for the example section would be helpful too.

Reply:
Regarding warning - will do.
Does the example for uzp should based on `http://lynxlynx.info/ie/modhub.php?Gibberlings3/SpellRevisions` even if the mod doesn't have them or I should use working example?

Reply:
A working one, of course, that's the point of an example — it can be tried successfully.

Reply:
done